pyccl/nl_pt/ept.py

```diff
@@ -131,6 +135,8 @@ class EulerianPTCalculator(CCLAutoRepr):
              documentation for more details.
         sub_lowk (:obj:`bool`): if ``True``, the small-scale white noise
              contribution to some of the terms will be subtracted.
+        usefptk (::obj:`bool`):) if `True``, will use the FASTPT IA k2
+        term instead of the CCL IA k2 term
```

Comment:
Why not require that people have the latest version of FastPT? My first reaction is that this adds unnecessary complexity that will need to be deprecated afterwards anyway (with the headache it entails, since it changes the API).

Reply:
I think there are two issues. One, for the basic k^2 term, we don't need FAST-PT, and we want the flexibility to choose the P_delta from the options CCL has access to. There may be future versions of this term that need some calculation to be done in FAST-PT.

Two, more generally, do we want to demand the latest (or fairly recent) FAST-PT for all CCL ept users? Probably good in the long term. Right now, FAST-PT is also changing rapidly, and is actually a bit behind in some functionality, so we don't expect all CCL users to actually have access to the bleeding edge.

Reply:
OK, I see, thanks @jablazek .

Then, how about:

1. Remove the `usefptk` option from this function.
2. Allow for `ak2_pk_kind` to take an additional option (e.g. `'pt_IA_k2'`), in which case the calculator uses the fast-pt prediction.

Then, if users select this option, you check if the fast-pt version allows for it and throw an error if it doesn't. My main aim here is not polluting the API with arguments that we may need to deprecate soon-ish, since they become a headache.

Reply:
As my message above, I've implemented what I meant in a new commit [here](https://github.com/LSSTDESC/CCL/pull/1137/commits/c551530432439df6ba00ade3159851bb0ff7be5f). Please take a look and feel free to disagree with me!